### PR TITLE
REPL examples in ES6

### PR DESCRIPTION
**Pull request by hemanth**
^

### CODE REVIEW

**Comment by branneman** [comment]
+1

**Comment by CrossEye** [comment]
This looks like a very good idea to me.  Anyone have objections?

File: repl/assets/js/examples.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ R.transduce(R.map(priceOrZero), R.add, 0, purchases);
                 title: "curry",
                 body: function () {
                     /*
-var f = R.curry(function(a, b, c) { return b + (a * c); });
+var f = R.curry((a, b, c) => b + (a * c) );
```
Comment (davidchambers) [inline repl/assets/js/examples.js]:
```diff
-var f = R.curry((a, b, c) => b + (a * c) );
+var f = R.curry((a, b, c) => b + (a * c));
```

Also, shall we remove the unnecessary parens wrapping `a * c`?

File: repl/assets/js/examples.js [hunk 2]
```diff
@@ -90,13 +90,13 @@ pipeEx([2, 3, 4, 5]);
                 title: 'useWith',
                 body: function() {
                     /*
-var sum = function() { 
-    return R.reduce(function(acc, x) { return acc + x; }, 0, arguments);
+var sum = function() {
```
Comment (davidchambers) [inline repl/assets/js/examples.js]: Could we use an arrow function here too? It's confusing to mix the two forms.

File: repl/assets/js/examples.js [hunk 3]
```diff
@@ -106,9 +106,9 @@ useWithEx(2, 3, 4);
                 title: 'converge',
                 body: function() {
                     /*
-var add = function(a, b) { return a + b; };
-var multiply = function(a, b) { return a * b; };
-var subtract = function(a, b) { return a - b; };
+var add = (a, b) => a + b;
+var multiply = (a, b) =>  a * b;
```
Comment (davidchambers) [inline repl/assets/js/examples.js]:
```diff
-var multiply = (a, b) =>  a * b;
+var multiply = (a, b) => a * b;
```

File: repl/assets/js/examples.js (hunk 1 above)
Comment (hemanth) [inline repl/assets/js/examples.js]: Will do that :)

File: repl/assets/js/examples.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ R.transduce(R.map(priceOrZero), R.add, 0, purchases);
                 title: "curry",
                 body: function () {
                     /*
-var f = R.curry(function(a, b, c) { return b + (a * c); });
+var f = R.curry((a, b, c) => b + a * c);
```
Comment (hemanth) [inline repl/assets/js/examples.js]: Looks ok without parens?

Comment (davidchambers) [inline repl/assets/js/examples.js]: :thumbsup:

File: repl/assets/js/examples.js [hunk 5]
```diff
@@ -90,13 +90,12 @@ pipeEx([2, 3, 4, 5]);
                 title: 'useWith',
                 body: function() {
                     /*
-var sum = function() { 
-    return R.reduce(function(acc, x) { return acc + x; }, 0, arguments);
-};
+var sum = () =>R.reduce((acc, x) => acc + x, 0, arguments);
```
Comment (davidchambers) [inline repl/assets/js/examples.js]:
```diff
-var sum = () =>R.reduce((acc, x) => acc + x, 0, arguments);
+var sum = () => R.reduce((acc, x) => acc + x, 0, arguments);
```

**Comment by hemanth** [comment]
@davidchambers done, thanks for the review.

**Comment by CrossEye** [comment]
:herb:

**Comment by hemanth** [comment]
:+1:

**Pull request merged**